tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1088,6 +1088,11 @@ test_i2c_tmp1075: tmp1075@98 {
 	interrupt-mode;
 };
 
+test_i2c_hs400x: hs400x@99 {
```

Comment:
please rebase on main: this should be `@9f`at the time of writing (and not sure why there is a bogus apds_9306 entry, looks like it was fixed at some point though https://github.com/zephyrproject-rtos/zephyr/commit/5be36eef47c006a4238400df79d6b667cc809b58#diff-e5e68be0d8d187414fd7e5acd16a07df950cace9e49bac77f4da6846dfe8a505R1106)

Reply:
Thank you, I've rebased on main (not sure why either, I worked from the 3.7 release)